Actions/ScheduleActions/ScheduleActionCreators.js

```diff
@@ -1,4 +1,4 @@
-import BASE_URL from 'TheaterSchedule/baseURL';
+import BASE_URL from '../../baseURL';
```

Comment:
avoid relative URL when you need go few times back

Reply:
what if I have different path?